src/components/ShowChooseCard.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ChooseCard from './ChooseCard';
+
+
+function ShowChooseCard() {
+  return (
+    <div className="App">
+      <ChooseCard image="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/130.png" name="Pikachu" type="https://www.pokepedia.fr/images/e/e5/Miniature_Type_Feu_JCC.png" pv='70'/>
```

Comment:
When naming components, prefer declarative names such as `CardSelection` or `CardSelectionPage`.

Reply:
Same for `ChooseCard` which could be `PokemonCard`.